### ISSUE TITLE
ProcessTiming and ItemGeometry from ClientId(0) will be skipped in TuiMonitor.item_geometry 

### ISSUE THREAD

**Issue by Kherrisan** [AFLplusplus/LibAFL#1771]
The trait ``Monitor`` will store ``ClientStats`` of each fuzzer client in a vector, and the index is the client id ``ClientId``. So when ``SimpleEventManager`` creates ``ClientStats`` with **client id 0** in ``handle_in_broker``：
```
monitor
    .client_stats_mut_for(ClientId(0))
    .update_user_stats(name.clone(), value.clone());
```
The new ``ClientStats`` will be stored in the **0-th** position of the vector. Unfortunately, **this position will be skipped by process_timing and item_geometry**.
```
fn process_timing(&mut self) -> ProcessTiming {
        let mut total_process_timing = ProcessTiming::new();
        total_process_timing.exec_speed = self.execs_per_sec_pretty();
        if self.client_stats.len() > 1 {
            ......
            for client in self.client_stats().iter().skip(1) {
                new_path_time = client.last_corpus_time.max(new_path_time);
                new_objectives_time = client.last_objective_time.max(new_objectives_time);
            }
            ......
        }
        ......
    }
```

```
fn item_geometry(&self) -> ItemGeometry {
        let mut total_item_geometry = ItemGeometry::new();
        if self.client_stats.len() < 2 {
            return total_item_geometry;
        }
        ......
        for client in self.client_stats().iter().skip(1) {
        ......
    }
```
The result is process_timing and item_geometry is all zero in TUI.

**Comment by tokatoka** [comment]
@ToSeven 
can you look at this?

**Comment by ToSeven** [comment]
Ok, let me take a look

**Comment by ToSeven** [comment]
The overall section of the TUI collects the statistics of all clients.  When `self.client_stats.len()` equals 1, it means the system has two clients: one broker and one client. Therefore, only one client has statistics.

**Comment by tokatoka** [comment]
so can I close it?

**Comment by domenukk** [comment]
The point is that `SimpleEventManager` doesn't have a broker, so client 0 is the important one (the only one with data)

**Comment by tokatoka** [comment]
@ToSeven can you fix this?

**Comment by domenukk** [comment]
Maybe each manager should have a fixed array of ignored ids? It'll be empty for `SimpleEventManager` and `0` for the other guys?

**Comment by domenukk** [comment]
Tried a fix in #2001
